README.md

```diff
@@ -170,6 +170,18 @@ For the [granite model above](https://huggingface.co/ibm-granite/granite-3.0-8b-
 
 The code internally uses [`DataCollatorForCompletionOnlyLM`](https://github.com/huggingface/trl/blob/main/trl/trainer/utils.py#L93) to perform masking of text ensuring model learns only on the `assistant` responses for both single and multi turn chat.
 
+Depending on various scenarios users might need to decide on how to use chat template with their data or which chat template to use for their use case.  
+
+Following are the Guidelines from us in a flow chart :  
+![guidelines for chat template](/docs/images/chat_template_guide.jpg)  
```

Comment:
@YashasviChaurasia can you update the path? use of root `/` would not work. You can use `docs/images/chat_template_guide.jpg`

Reply:
Sure, Thank you